src/Microsoft.OData.Core/JsonLight/ODataJsonLightReader.cs

```diff
@@ -399,15 +416,6 @@ protected override bool ReadAtNestedPropertyInfoImplementation()
             return this.ReadAtNestedPropertyInfoSynchronously();
         }
 
-        /// <summary>

```

Comment:
No need for this since the implementation in the base class mirrors this, i.e.,
```csharp
/// <summary>
/// Implementation of the reader logic when in state 'PropertyInfo'.
/// </summary>
/// <returns>A task which returns true if more items can be read from the reader; otherwise false.</returns>
protected virtual Task<bool> ReadAtNestedPropertyInfoImplementationAsync()
{
    return TaskUtils.GetTaskForSynchronousOperation<bool>(this.ReadAtNestedPropertyInfoImplementation);
}
```